services/QuillLMS/lib/tasks/add_grammar_api_rules_and_feedback.rake

```diff
@@ -0,0 +1,44 @@
+namespace :grammar_api_rules_and_feedback do
+    desc 'data migration for grammar-api specific rules and feedback'
+    task :insert => :environment do
+        rules_csv = CSV.parse(File.read('lib/data/rules.csv'), headers: true)
+        valid_rules = rules_csv.select do |r| 
+            ['Grammar API', 'Opinion API'].include?(r['Module']) && r['Rule UID'].present? 
+        end
+
+        ActiveRecord::Base.transaction do 
+            valid_rules.each do |r|
+                created_rule = Comprehension::Rule.find_or_initialize_by(uid: r['Rule UID'])
+                created_rule.attributes = {
+                    name: r['Rule'],
+                    description: r['Rule Description'],
+                    universal: true,
+                    rule_type: r['Module'] == 'Grammar API' ? 'grammar' : 'opinion',
+                    optimal: false,
+                    suborder: r['Overall Rule Priority'],
+                    state: 'active',
+                }
+                created_rule.save!
+
+                # rubocop:disable all
```

Comment:
Rubocop throws Layout/AssignmentIndentation: Indent the first line of the right-hand-side of a multi-line assignment.

even though the line seems indented correctly